pkg/controller/appdefinition/volume.go

```diff
@@ -206,6 +209,55 @@ func toPVCs(req router.Request, appInstance *v1.AppInstance) (result []kclient.O
 	return
 }
 
+func getPVForVolumeBinding(req router.Request, appInstance *v1.AppInstance, binding v1.VolumeBinding) (*corev1.PersistentVolume, error) {
```

Comment:
Does it makes sense for this to be a public function? I ask because I have a use case where I would use this exact function in a different controller I'm writing.

Reply:
Or rather, I would use the "detect if this binding will bind to an existing volume" logic.

Reply:
It's public now